base/shell/explorer/trayntfy.cpp

```diff
@@ -37,14 +39,19 @@ class CTrayNotifyWnd :
     public IOleWindow
 {
     CComPtr<IUnknown> m_clock;
+public:
```

Comment:
Isn't it possible to maintain `m_ShowDesktopButton` in private?

Reply:
Huh, seems it is...will do :+1: